temporal-testing/src/main/java/io/temporal/internal/testservice/TestWorkflowService.java

```diff
@@ -222,21 +255,38 @@ private TestWorkflowService(boolean isOutOfProc, int port) {
   @Override
   public void close() {
     log.debug("Shutting down TestWorkflowService");
-    client.close();
+
+    if (outOfProcessServer != null) {
+      log.info("Shutting down out-of-process GRPC server");
+      outOfProcessServer.shutdown();
+    }
+
+    if (inProcessServer != null) {
+      log.info("Shutting down in-process GRPC server");
+      inProcessServer.shutdown();
+      client.channel.shutdown();
```

Comment:
Can we bring `client.shutdown()` back? It also shuts down stubs internally.

Reply:
Oof, that was a bad merge from our local copy. Thanks for catching that.